sdk/attestation/azure-security-attestation/test/ut/tpmattestation_test.cpp

```diff
@@ -0,0 +1,118 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include "attestation_collateral.hpp"
+#include "azure/attestation/attestation_administration_client.hpp"
+#include "azure/attestation/attestation_client.hpp"
+#include <azure/core/internal/json/json.hpp>
+#include <azure/core/test/test_base.hpp>
+#include <azure/identity/client_secret_credential.hpp>
+#include <gtest/gtest.h>
+#include <tuple>
+
+using namespace Azure::Security::Attestation;
+using namespace Azure::Security::Attestation::Models;
+using namespace Azure::Core;
+
+namespace Azure { namespace Security { namespace Attestation { namespace Test {
+
+  enum class InstanceType
+  {
+    Shared,
+    AAD,
+    Isolated
+  };
+
+  // cspell: words aikcert
```

Comment:
Also can be placed in cspell.json

Reply:
aikcert is highly specific to the TPM attestation API and probably shouldn't go into a global cspell.json file.